lib/hanami/view/escape.rb

```diff
@@ -165,10 +165,13 @@ def self.extended(base)
       # @since 0.4.0
       # @api private
       def method_added(method_name)
+        visibility = :private if private_method_defined? method_name
+        visibility ||= :protected if protected_method_defined? method_name
```

Comment:
I'm not really proud of it, but it goes as this: if visibility was set to anything truthy before, skip the assignment. That being said, the first assignment could also be a or_equals `||=` but it ended up even more cryptic. =(

I'm all open to suggestions on a better way to do that (I haven't found one, tho).